src/main/scala/com/cloudera/finance/examples/HistoricalValueAtRiskExample.scala

```diff
@@ -47,8 +47,8 @@ object HistoricalValueAtRiskExample {
     def loadTS(inputDir: String, lower: DateTime, upper: DateTime): TimeSeriesRDD = {
       val histories = YahooParser.yahooFiles(inputDir, sc)
       histories.cache()
-      val start = histories.map(_.index.first).takeOrdered(1).head
-      val end = histories.map(_.index.last).top(1).head
+      val start = histories.map(_.index.first()).takeOrdered(1).head
```

Comment:
Ah ok.  That always seemed weird to me because it falls apart when you start adding args, but that's a different battle.

In that case I think we should make `first` and `last` no-arg methods.